python/turbine_models/custom_models/compile_hf_transformer_model.py

```diff
@@ -0,0 +1,124 @@
+"""
+
+
+
+# TODO (Dan): replace this with a file once I figure out paths on windows exe
```

Comment:
This needs to not exist more than once in the whole codebase, and could come from its own python file

Reply:
(as opposed to json file, which is harder to bundle in an exe)